src/discof/repair/fd_repair_tile.c

```diff
@@ -1439,6 +1444,13 @@ unprivileged_init( fd_topo_t *      topo,
 
   fd_repair_update_addr( ctx->repair, &ctx->repair_intake_addr, &ctx->repair_serve_addr );
 
+  /* TODO: this is a hack so replay generates votes.
+     Once we develop a better notion of 'caught up',
+     remove all uses of current_slot in this file */
```

Comment:
```suggestion
  /* TODO: this is a hack to set the first turbine slot so that replay knows when we have caught up. This is important for voting, because you need have "read-back" your latest landed tower before you can vote. Using an fseq is a temporary hack, and this will be replaced with the standard frag-signaling pattern across tiles with a separate consensus tile. */
```

Reply:
(reformat to 72 cols)